src/FluentMigrator/Builders/Schema/Sequence/SchemaSyntaxQuery.cs

```diff
@@ -0,0 +1,49 @@
+﻿#region License
```

Comment:
Isnt this file supposed to be named SchemaSequenceQuery?

Reply:
Yes. I am renaming it.